tests/providers/amazon/aws/sensors/test_rds.py

```diff
@@ -50,15 +51,18 @@
 EXPORT_TASK_SOURCE = 'arn:aws:rds:es-east-1::snapshot:my-db-instance-snap'
 
 
-def _create_db_instance_snapshot(hook: RdsHook):
+def _create_db_instance(hook: RdsHook):
     hook.conn.create_db_instance(
         DBInstanceIdentifier=DB_INSTANCE_NAME,
-        DBInstanceClass='db.m4.large',
-        Engine='postgres',
+        DBInstanceClass="db.t4g.micro",
```

Comment:
I took the liberty of changing the db instance class to the smallest possible size here and in the system test.

Reply:
Have you tested this?  The 'large' is the smallest option on the AWS console when creating a new RDS instance.  In fact, I just checked to verify that and the m4 generation isn't even an option in their dropdown anymore, it's only gen 5 or gen 6 now.  I just want to make sure it'll run alright with the lower specs.  I know the RDS Export system test takes about 20 minutes to run on the `large`, I'm not sure how much longer this is going to add.

Reply:
Yes I ran the system tests. I don't remember it taking any longer than 20 minutes.

Try changing the *Availability and durability* in your console to "single instance" and you should be able to select db.t4g.micro, which is the least expensive instance type.

Reply:
Alright, then I'm cool with the change.  Thanks for checking that.